tests/test_balance.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2021 eprbell
```

Comment:
The convention we've been following is that new files have the creator in the copyright line: 
```suggestion
# Copyright 2024 qwhelan
```

Reply:
Done